### DISCUSSION TITLE
my school recently blocked running files is there anyway to get around this

### DISCUSSION THREAD

**Discussion by buddyted** [3kh0/ext-remover#977] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how do I

**Comment by Solar-Geaux** [comment]
a little late but, are you talking about html files or straight up files like rar/zip. if you want help please give me a screenshot there's probably a way to fix SOME of the blocked files. you obviously wont be able to run whole applications.

**Reply by buddyted** [reply]
Any

**Comment by 3kh0** [comment]
Running what files??

**Reply by buddyted** [reply]
html

**Comment by LizzyEll** [comment]
Like this? As in file paths?
![Screenshot 2024-04-08 10 30 56 AM](https://github.com/3kh0/ext-remover/assets/92237193/bc6749cf-f661-4371-828c-7d0dd1801cce)

**Reply by buddyted** [reply]
Yes is there anyway to get past that

**Reply by LizzyEll** [reply]
not as of now. For websites policy blocked (like this site) you can easiler bypass a block such as `https://github.com/3kh0/ext-remover` by simply adding an extra forward slash, `https://github.com/3kh0//ext-remover`. I don't know however how it would work on an entire url scheme.

**Reply by yuTool666** [reply]
https://www.programiz.com/html/online-compiler/
のようなサイトにコードを書いてみましょう

**Comment by Blobby-Boi** [comment]
If it’s an html file just make a data: link for it

**Reply by LizzyEll** [reply]
true!
In case you aren't aware of this scheme @buddyted...
`data:text/html,<html data>`
I'm planning on starting a project completely in plain text in which it is a game site create entirly with a single `data:` URL.

**Reply by AshtonDavies** [reply]
Extensions can track statistics of data URLs, so don't get too excited.

**Reply by LizzyEll** [reply]
Yeah, but in my case I have a nameserver that blocks the filter extension (GoGuardian/CK-Express) from installing and if it does install it can't connect to the internet. The extension cannot send any data to the administrators haha

**Reply by AshtonDavies** [reply]
That's not solving it for everyone you share the site with, so it can still get seen and blocked.

Nice to know you have a little knock off cauDNS.

**Reply by LizzyEll** [reply]
It would only really be meant for a close group of people I know and/or I could create a repo with instructions on setting up a simple DNS. I could also request an additional Static IP for my server and host a public nameserver there for anyone who wants to use it. rn im literally just using pihole on a linode server lmao

**Reply by AshtonDavies** [reply]
What I would recommend is working on an automatic about:blank system for your site whatever way you end up configuring it in terms of the URL.

I haven't heard of Linode before. Is it good for static or manual DNS hosting?

**Reply by LizzyEll** [reply]
It isn't specifically for DNS. They just have really cheap rates for servers and support Kubernetes/Docker images. They also have a vast variety of server locations 
Also, I'm not sure why it doesn't work, but `about:blank` exploits don't work on our chromebooks.

**Reply by LizzyEll** [reply]
@AshtonDavies do you have a contact email? I would like to ask you some questions about a separate exploit I was looking to develop.

**Reply by AshtonDavies** [reply]
> @AshtonDavies do you have a contact email? I would like to ask you some questions about a separate exploit I was looking to develop.

All in my profile. I prefer Discord if that works for you, but you can still email me.

**Reply by buddyted** [reply]
hay how do a add it in the link and were

**Reply by Blobby-Boi** [reply]
You can use my [data URL generator](https://github.com/Blobby-Boi/data-URL-generator)

**Reply by md4r2** [reply]
bit late but u might wanna try using a simple online html editor like [this](https://www.tutorialspoint.com/online_html_editor.php)

**Comment by Inglan2** [comment]
upload to a server or data url

**Reply by buddyted** [reply]
How

**Reply by Inglan2** [reply, marked as answer]
data url. upload HTML then press copy and paste into new tab: https://dataurl.app/

github as server. watch until end of chapter: https://youtu.be/EXfFBEuCAr0?t=246

w3 schools tryit editor: https://www.w3schools.com/html/tryit.asp?filename=tryhtml_basic
